<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>1183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>672</width>
      <height>980</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;Arial&quot;;</string>
    </property>
    <property name="title">
     <string>    股票查询系统</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_4">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>数据准备</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QProgressBar" name="progressBar">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Arial&quot;;</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Arial&quot;;</string>
       </property>
       <property name="title">
        <string>                请输入需要查询的股票信息</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton">
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QPlainTextEdit" name="plainTextEdit_3">
          <property name="styleSheet">
           <string notr="true">font: 11pt &quot;Arial&quot;;</string>
          </property>
          <property name="plainText">
           <string>请输入真实存在的股票信息。
股票名称格式示例：&quot;000001.SZ&quot;
股票年月格式示例：&quot;200101&quot;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_3">
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>预测</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="labelX">
              <property name="styleSheet">
               <string notr="true">font: 10pt &quot;Arial&quot;;
font: 11pt &quot;Arial&quot;;</string>
              </property>
              <property name="text">
               <string>股票名称</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="LineX"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="1">
             <widget class="QLineEdit" name="LineY"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="labelY">
              <property name="styleSheet">
               <string notr="true">font: 10pt &quot;Agency FB&quot;;
font: 11pt &quot;Arial&quot;;</string>
              </property>
              <property name="text">
               <string>股票年月</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_3"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Arial&quot;;</string>
       </property>
       <property name="title">
        <string>              请输入需要查询的股票月份</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QFormLayout" name="formLayout_5">
            <item row="0" column="0">
             <widget class="QLabel" name="labelYear">
              <property name="styleSheet">
               <string notr="true">font: 10pt &quot;Arial&quot;;
font: 12pt &quot;Arial&quot;;</string>
              </property>
              <property name="text">
               <string>年</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="LineYear"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="labelMonth">
              <property name="text">
               <string>月</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="LineMonth"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QPlainTextEdit" name="plainTextEdit_2">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Arial&quot;;</string>
          </property>
          <property name="plainText">
           <string>输入指定年月后返回该月份sharpe指数前30的股票
输入格式：
年：2000 - 2021
月：1 - 12</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_2">
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>动画显示</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>747</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
